src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
```

Comment:
It is a good practice to have a parent class for all controlles, for example a class "ApiController", where you have everything that is the same for all classes (be they methods, be they implementations, etc.)